webrtc.html

```diff
@@ -1873,28 +1800,100 @@ <h4>Set the RTCSessionDescription</h4>
                     </ol>
                   </li>
                   <li>
-                    <p>If we are processing a rollback then for each
-                    <var>transceiver</var> in <var>connection</var>'s <a
-                    href="#transceivers-set">set of transceivers</a>, run the
-                    following steps:</p>
+                    <p>Otherwise, (if <var>description</var> is of type <code>"rollback"</code>)
+                    run the following steps:</p>
                     <ol>
                       <li>
-                        <p>Set <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\SenderTransport]]</a>
-                        to <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\StableStateSenderTransport]]</a>
-                        and set <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\StableStateSenderTransport]]</a>
```

Comment:
> 1. Since these are backup slots, s/StableStateSenderTransport/LastStableSenderTransport/

Done. Also s/StableStateReceiveCodecs/LastStableStateReceiveCodecs.

> 2. Since the algorithm is getting so long in #2384, skip setting them to null.

Done.